@kamoltat - I am encountering the following error while creating the Shaman build.After applying the fix, please add the `needs-qa` label.


```
--- pr 68365 --- pulling https://github.com/kamoltat/ceph.git branch wip-ksirivad-fix-75418
remote: Enumerating objects: 11, done.
remote: Counting objects: 100% (9/9), done.
remote: Total 11 (delta 9), reused 9 (delta 9), pack-reused 2 (from 1)
Unpacking objects: 100% (11/11), 1.44 KiB | 735.00 KiB/s, done.
From https://github.com/kamoltat/ceph
 * branch                    wip-ksirivad-fix-75418 -> FETCH_HEAD
Auto-merging qa/suites/upgrade/squid-x/stress-split/1-start.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/squid-x/stress-split/1-start.yaml
Auto-merging qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
CONFLICT (content): Merge conflict in qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
Automatic merge failed; fix conflicts and then commit the result.
Unable to access https://github.com/kamoltat/ceph.git, retrying..
error: Pulling is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.
Traceback (most recent call last):
  File "/home/skanta/Office/Project/Upstream/wip-bharath17-testing/src/script/build-integration-branch", line 193, in <module>
    main()
  File "/home/skanta/Office/Project/Upstream/wip-bharath17-testing/src/script/build-integration-branch", line 175, in main
    raise Exception(message)
Exception: Unable to resolve conflict when merging PR#68365
```

--- a/qa/suites/upgrade/squid-x/stress-split/1-start.yaml
+++ b/qa/suites/upgrade/squid-x/stress-split/1-start.yaml
@@ -31,6 +31,7 @@ overrides:
       - OBJECT_UNFOUND
       - pg .* is stuck peering
       - OSD_HOST_DOWN
+      - PG_DAMAGED
 tasks:
 - install:
     branch: squid

--- a/qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
+++ b/qa/suites/upgrade/tentacle-x/stress-split/1-start.yaml
@@ -30,6 +30,7 @@ overrides:
       - osd down
       - OBJECT_UNFOUND
       - OSD_HOST_DOWN
+      - PG_DAMAGED
 tasks:
 - install:
     branch: tentacle

